packages/nimble-components/src/select/index.ts

```diff
@@ -51,6 +50,10 @@ declare global {
 // eslint-disable-next-line @typescript-eslint/no-invalid-void-type
 type BooleanOrVoid = boolean | void;
 
+const isListOption = (el: Element): el is ListOption => {
+    return el instanceof ListOption;
+};
```

Comment:
Can we use that `isListboxOption` from foundation like we are in the template?

If we are looking for s stricter check with our nimble specific apis maybe call it `isNimbleListOption`?